tools/ratbagctl.in.in

```diff
@@ -194,15 +194,20 @@ def print_profile(d, p, level):
                                                        " (disabled)" if not p.enabled else "",
                                                        " (active)" if p.is_active else ""))
     if p.enabled:
-        print(" " * level + "Name: {}".format(p.name or 'n/a'))
-        print(" " * level + "Report Rate: {}Hz".format(p.report_rate))
-        print(" " * level + "Resolutions:")
-        for r in p.resolutions:
-            print_resolution(d, p, r, level + 2)
-        for b in p.buttons:
-            print_button(d, p, b, level)
-        for l in p.leds:
-            print_led(d, p, l, level)
+        if p.name:
+            print(" " * level + "Name: {}".format(p.name))
```

Comment:
@whot, is this fine? Or do you want to keep the old behavior for the profile name?

Reply:
I ... don't know. I still consider `ratbagctl` a debugging tool, so having something printed but empty shows we support it but have no data. Which is often better during debugging than wondering why something is altogether missing.

There's no single correct answer here either but tbh given the time I have to invest in libratbag I won't push any personal preferences here.

Reply:
I get your point. I mainly see it as a user tool. And as such I think it's ugly to have something like
```
  Report Rate: 0Hz
  Resolutions:
```
I think both POVs are valid. I will let @bentiss weight in before making a final decision.

Reply:
I think we should keep the name.
`Name: n/a` means *the profile name can not be stored on the device*
which is different from `Name: ` (empty name) where the name is actually `\0`

I'm fine removing the `Report Rate` and `Resolution` for devices without these capabilities.

Reply:
> I think we should keep the name.
Name: n/a means the profile name can not be stored on the device
which is different from Name: (empty name) where the name is actually \0

Not entirely true. For ex. Logitech `0x8100` devices support storing the profile name but they show up as `n/a` if there's no valid name. I am not sure having the profile name as `\0` has the same behavior.

Reply:
Just to give my two cents here, if the name is empty, there is little reason to show it anyway. But if we can't agree here, can we just keep the old behavior and merge this? Think it would be a nice addition for the next release with the initial keyboard support.

Reply:
I will keep the name.